--- a/drivers/net/wireless/realtek/rtw88/phy.c
+++ b/drivers/net/wireless/realtek/rtw88/phy.c
@@ -764,8 +764,8 @@ void rtw_phy_dynamic_mechanism(struct rt
 {
 	/* for further calculation */
 	rtw_phy_statistics(rtwdev);
-	rtw_phy_dig(rtwdev);
-	rtw_phy_cck_pd(rtwdev);
+//	rtw_phy_dig(rtwdev);
+//	rtw_phy_cck_pd(rtwdev);
 	rtw_phy_ra_track(rtwdev);
 	rtw_phy_tx_path_diversity(rtwdev);
 	rtw_phy_cfo_track(rtwdev);
